react/index.html

```diff
@@ -1,13 +1,49 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React</title>
+    <title>webchat</title>
   </head>
   <body>
     <div id="root"></div>
     <script type="module" src="/src/main.jsx"></script>
+    <script>
+      (function () {
+        var PLATFORM_ID = "%VITE_REACT_APP_PLATFORM_ID%",
```

Comment:
Do the environment variables require VITE_REACT_APP prefix? 
Can it be `HS_PLATFORM_ID` and `HS_DOMAIN` instead?

Reply:
Yes it's important to write `VITE_REACT_APP` prefix 
I can change it to `VITE_REACT_APP_HS_PLATFORM_ID` and `VITE_REACT_APP_HS_PLATFORM_ID` ?